lib/controllers/skill-metadata-controller/index.js

```diff
@@ -111,6 +111,10 @@ module.exports = class SkillMetadataController {
    * Validates domain info
    */
   validateDomain() {
+    if (process.env.ASK_SKIP_DOMAIN_VALIDATION) {
```

Comment:
any thought on `ask deploy --skipDomainValidation`? Is that too much work to plum through to SkillMetadataController?

Reply:
command line arguments are for features we want to have customers use / more frequently.   This seems more of an edge case that we need to enable a skill that doesn't match custom or music.  🤷🏼‍♂️  I can plumb it through as a argument if that's the decision.  might need to get UX involved for sign off.